### PR TITLE
[4.1] Remove invalid tooltip option in media field

### PR DESCRIPTION
**Pull request by brianteeman**
This PR removes the non-existent tooltip option

pr for #36938

### CODE REVIEW

**Comment by laoneo** [comment]
Thanks @brianteeman. Are the other two options still working?

**Comment by brianteeman** [comment]
of course

**Comment by tecpromotion** [comment]
I have tested this item :white_check_mark: successfully on 936ed17c01be192aba854c88c65d1995eacd25b6<br /><br />database value for fieldparams before patch
`{"directory":"","preview":"tooltip","image_class":""}`

database value for fieldparams after apply patch
`{"directory":"","preview":"tooltip","image_class":""}`

database value for fieldparams after save the field
`{"directory":"","preview":"","image_class":""}`

This should fit, because even without saving the field again, the value "tooltip" has no effect.<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/37115">issues.joomla.org/tracker/joomla-cms/37115</a>.</sub>

File: plugins/fields/media/media.xml [hunk 1]
```diff
@@ -35,10 +35,9 @@
 					type="list"
 					label="PLG_FIELDS_MEDIA_PARAMS_PREVIEW_LABEL"
 					class="list"
-					default="tooltip"
+					default="true"
 					validate="options"
 					>
-					<option value="tooltip">PLG_FIELDS_MEDIA_PARAMS_PREVIEW_TOOLTIP</option>
```
Comment (Quy) [inline plugins/fields/media/media.xml:41]: Add `// Deprecated parameter value`?

https://github.com/joomla/joomla-cms/blob/4408a5f183e9041af92692f236588323630f0bbf/layouts/joomla/form/field/media.php#L70

**Comment by pabloarias** [comment]
I have tested this item :white_check_mark: successfully on 936ed17c01be192aba854c88c65d1995eacd25b6<br /><br />Tested in Joomla 4.1.1-dev with PHP 8.0.15.

Tooltip option was removed. Inline and No options work fine.

Thanks<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/37115">issues.joomla.org/tracker/joomla-cms/37115</a>.</sub>

**Comment by laoneo** [comment]
@brianteeman do you want to add the comment, mentioned by @Quy?

**Comment by brianteeman** [comment]
I am happy with the PR as it is

**Comment by laoneo** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/37115">issues.joomla.org/tracker/joomla-cms/37115</a>.</sub>

**Comment by Quy** [comment]
Thanks

**Pull request merged**